release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1699,6 +1699,12 @@ Kubernetes 1.29 removed the following deprecated APIs, so you must migrate manif
 
 * Previously, the Bare Metal Operator (BMO) was not using a leader lock to control incoming and outgoing Operator pod traffic. After an OpenShift `Deployment` object included a new Operator pod, the new pod competed with system resources, such as the `ClusterOperator` status, and this terminated any outgoing Operator pods. This issue also impacted clusters that do not include any bare-metal nodes. With this release, the BMO includes a leader lock to manage new pod traffic, and this fix resolves the competing pod issue. (link:https://issues.redhat.com/browse/OCPBUGS-25766[*OCPBUGS-25766*])
 
+* Previously, when you attempted to delete a `BareMetalHost` object before the installation started, the metal3 Operator attempted to create a `PreprovImage` image. The process of creating this image caused the `BareMetalHost` object to still exist in certain processes. With this release, an exception is added for this situation so that the `BareMetalHost` object is deleted without impacting running processes. (link:https://issues.redhat.com/browse/OCPBUGS-33048[*OCPBUGS-33048*])
+
+* Previously, a Redfish virtual media Hewlett Packard Enterprise (HPE) integrated Lights Out (iLO) 5 bare-metal machine's compression was forcibly disabled to workaround other unrelated issues in different hardware models. This caused the `FirmwareSchema` resource to be missing from each iLO 5 bare-metal machine. Each machine needs compression to fetch message registries from their Redfish Baseboard Management Controller (BMC) endpoints. With this release, each iLO 5 bare-metal machine that needs the `FirmwareSchema` resource does not have compression forcibly disabled. (link:https://issues.redhat.com/browse/OCPBUGS-31104[*OCPBUGS-31104*])
```

Comment:
I think bare metal is also just two words without a hyphen:
```suggestion
* Previously, a Redfish virtual media Hewlett Packard Enterprise (HPE) integrated Lights Out (iLO) 5 bare metal machine's compression was forcibly disabled to workaround other unrelated issues in different hardware models. This caused the `FirmwareSchema` resource to be missing from each iLO 5 bare metal machine. Each machine needs compression to fetch message registries from their Redfish Baseboard Management Controller (BMC) endpoints. With this release, each iLO 5 bare metal machine that needs the `FirmwareSchema` resource does not have compression forcibly disabled. (link:https://issues.redhat.com/browse/OCPBUGS-31104[*OCPBUGS-31104*])
```

Reply:
![Screenshot from 2024-06-24 09-34-02](https://github.com/openshift/openshift-docs/assets/57954076/5387ca3b-d2a8-4cad-ae2e-a80f4608b854)